loops-and-arrays.js

```diff
@@ -6,6 +6,18 @@
 
 const names = ["Henk", "Piet", "Fred", "Joop"];
 
+console.log("originele waardes:");
+for (let i = 0; i < names.length; i++) {
+    console.log(names[i]);
+}
+
+console.log("aangepaste waardes:");
+for (let i = 0; i < names.length; i++) {
+    const nickName = names[i];
```

Comment:
Je maakt gebruik van een additionele variabele, wat uiteraard prima werkt. Het zou wel onoverzichtelijk kunnen worden qua aantal variabelen in grotere stukken code. Waarbij je je afvraagt wat een bepaalde variabele doet, terwijl het alleen maar een 'tussenvariabele' is.